etc/docker/dev/docker-compose-kafka.yml

```diff
@@ -0,0 +1,440 @@
+version: "3"
```

Comment:
Does this file get used anywhere?  If not, can it be removed?

Reply:
I use this file for internal testing